snuba/clickhouse/formatter.py

```diff
@@ -0,0 +1,95 @@
+from typing import Optional, Sequence
+from snuba.query.expressions import (
+    Column,
+    CurriedFunctionCall,
+    Expression,
+    ExpressionVisitor,
+    FunctionCall,
+    Literal,
+)
+from snuba.query.parsing import ParsingContext
+from snuba.util import escape_alias, escape_col, escape_string
+
+# Tokens used when formatting. Defining them as costant
+# will make it easy (if/when needed) to make some changes
+# to the layout of the output.
+NULL = "NULL"
+COMMA = ","
+SPACE = " "
+TRUE = "true"
+FALSE = "false"
+QUOTE = "'"
+OPEN_BRACKET = "("
+CLOSED_BRACKET = ")"
```

Comment:
These are parentheses, not brackets (which would be `[` and `]`.)

Reply:
mhmh, I knew my english teacher in primary school was lying about a lot of things.